### DISCUSSION TITLE
How To dump or print the query ?

### DISCUSSION THREAD

**Discussion by JenuelDev** [jenssegers/laravel-mongodb#2155] [Q&A] [answered]
I have made queries in my model, but I just want to dump the query to make sure the query is right in MongoDB? Is there any function to do that?

**Comment by albertcito** [comment]
I try to do `dd($email->dump());` but it fails:

> This method is not supported by MongoDB. Try "toMql()" instead.

You can use it: 

```php
\Illuminate\Support\Facades\DB::connection('mongo')->enableQueryLog();
$email = MongoEmail::where('id', 'your_value_here');
$email->get();
$queries = \Illuminate\Support\Facades\DB::connection('mongo')->getQueryLog();
dd($queries);
```

**Comment by GromNaN** [comment, marked as answer]
The `Query\Builder::dump()` method will be fixed by https://github.com/mongodb/laravel-mongodb/pull/2727
